www/src/content/docs/form/reference.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Config Reference 
+description: Reference for simple-stack-form
+sidebar:
+  order: 4
+---
+
+## `injectMiddleware`
+
+**Type:** `true | false`
+
+The adapter option `injectMiddleware` allows to opt-out of the default behaviour, which injects a middleware. This could be useful if you want to add your own data retrivial and `POST` handling logic.
```

Comment:
```suggestion
The adapter option `injectMiddleware` allows you to opt-out of middleware injection used to expose [`Astro.locals.form`](form/parse/#astrolocalsform). This could be useful if you want to add your own data retrieval and `POST` handling logic.
```